modules/core/src/main/scala/com.snowplowanalytics.snowplow.databricks/processing/DatabricksUploader.scala

```diff
@@ -123,6 +124,21 @@ object DatabricksUploader {
     case nf: NotFound =>
       // NotFound exception messages are clean, short and helpful
       s"Not Found: ${nf.getMessage}"
+    case e: UnknownHostException =>
+      if (e.getMessage.matches(""".*\bnodename nor servname provided, or not known\b.*"""))
+        "Invalid Databricks hostname"
+      else
+        e.getMessage
+    case e: NullPointerException =>
```

Comment:
This really annoys me that we have to catch a `NullPointerException`.  And again match on a woolly message which I fear could change format in the future.  But I don't have a suggested alternative.

Reply:
Yeah, agree 🫤